src/utils/index.js

```diff
@@ -1,3 +1,19 @@
+import amplitude from "amplitude-js";
+
+export const setupAmplitude = () => {
+  if (typeof window === "undefined") {
+    return;
+  }
+
+  amplitude.getInstance().init("656f6573a88e63d3dc2ce32e33ef8284");
```

Comment:
This will now use production amplitude key for all envs. We should start using .envs files. You could have a look at https://www.gatsbyjs.org/docs/environment-variables/, or if you need help I'm happy to help.